rollback1.py

```diff
@@ -8,15 +8,28 @@
 
 
 class Rollback1TestCase(unittest.TestCase):
-    
+
+    def setUp(self):
+        global url, prKey, token, waitTx, host, db, pas, login
+        self.conf = config.readMainConfig()
+        url = self.conf["url"]
```

Comment:
what the point of all that variables, i think it's better just readMainConfig(), and use self.conf everywhere